src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments = [] }) => (
+  <div className="CommentList">
+    {comments.length > 0 ? (
+      comments.map(comment => (
+        <CommentInfo comment={comment} key={comment.id} />
+      ))
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    )}
```

Comment:
`CommentList` components should only render the list of comments. The rest of the logic should be described in parent component e.g.
```
comments.length > 0 ? <CommentList /> : something else
```
This `something else` should be `<hr /> + <b>No comments yet</b>`

Reply:
The same here, if I got it right.